source/less/blocks/main-container.less

```
@@ -15,23 +22,26 @@
   width: 1200px;
   height: 57px;
   position: absolute;
+  bottom: 0%;
+  object-fit: cover;
+  margin-left: -300px;
```

```suggestion
  background-position: center;
```
object-fit не очень уместен для фона, а эта граница у тебя задана как фон в псевдоэлементе.
Управлять расположением через margin на каждой ширине - тоже кажется ошибочным, он всегда должен быть по центру блока, а не точно выставлен для конкретных разрешений